extensions/ql-vscode/src/quick-query.ts

```diff
@@ -158,7 +160,13 @@ export async function displayQuickQuery(
   }
 }
 
-async function checkShouldRewrite(qlPackFile: string, newDependency: string) {
+async function checkShouldRewrite(
+  qlPackFile: string | undefined,
+  newDependency: string,
+) {
+  if (!qlPackFile) {
+    return true;
+  }
```

Comment:
I see we don't have a test file for `quick-query` functions so this is a stretch. If the setup isn't terribly complicated, do you think it's worth adding one to test what this is doing? 

I do see we have a `queries.test.ts` file which I guess covers some of the quick-query functionality.

Reply:
I could add a test for this function, but I don't really know what input values it would actually receive. I'm not quite sure when the function should be returning `true` or `false` or what it's purpose is, so I'm hesitant to add tests based purely on its implementation, rather than on its intended behaviour.

Reply:
Quick query is a feature where the IDE generates a bare-bones qlpack in a temporary directory that is tailored to the current language. It is meant to be semi-ephemeral.

When you call the quick query command, the extension checks if the query query pack exists already and if it is for the correct language of the current database.